### PR TITLE
Update the -EnableHTTPRedirect section

### PR DESCRIPTION
**Pull request by piaudonn**
Using the parameter does not dynamically open the port 80. This change it to reflect this and provide an example in PowerShell.

### CODE REVIEW

**Comment by e0i** [comment]
@piaudonn Hello, Would you kindly change the target branch of the PR from `live` to `master` in order to proceed with the review? Thank you for your contributions.

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
The example needs to be moved outside of the switch description into the examples section. Once the example is moved, no changes need to be made to the switch description itself.

**Comment by get-itips** [comment]
Agree with @tseward

**Comment by dariomws306** [comment]
@piaudonn, if you create a web app publish using Add-WebApplicationProxyApplication without -EnableHTTPRedirect, Does it create the http or https rule in the firewall automatically?

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

FYI: #1524 is a Sister PR applying the same changes to another document.

**Comment by piaudonn** [comment]
@dariomws306 no it doesn't. I had a couple of customer complaining about the fact we have not documented it that way anywhere. So I figured these two pages should be the occasion. I feel like even if we move the cmdlet example later, the description should still mentions that there is no rule by default. Else I don't see how we can even introduce the example later on. Also, note that the example is not an example on how to use the documented cmdlet but an example on how to enable a firewall rule. For the latter, I am not sure if the example section would be a good fit. Maybe a note section with a comment that the rule isn't enabled?

**Comment by dariomws306** [comment]
@piaudonn thank you. So I think a note should be added to the cmdlet description to warn about that, not in the EnableHTTPRedirect parameter.
And maybe we can avoid the New-NetFirewallRule example, as this doesn't correspond to this cmdlet.

What do you all think?

**Comment by tseward** [comment]
I'm in agreement, @dariomws306

**Comment by e0i** [comment]
@piaudonn Any updates with revision suggestions provided by the reviewers? Thanks.

**Comment by piaudonn** [comment]
Nope I'm in agreement. Should I update the document in this pull request?

**Comment by e0i** [comment]
@dariomws306 Any comments? 

> Nope I'm in agreement. Should I update the document in this pull request?
> It would just be this in the parameter description:
> 
> > Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication.
> 
> No mention of the Net-NetFirewallRule example. Althought, maybe we can link to the Windows Firewall documentation? I feel like there is a risks that customer end up creating a too permissive rule or worst, disable the firewall all together.

**Comment by dariomws306** [comment]
Hi @piaudonn, yes please modify the document in the same pull request. Thanks!

**Comment by e0i** [comment]
Hello, @piaudonn As with PR #1524, We would like to proceed with merge upon your follow-up with reviewers' suggestions. Thank you again for your patience with the process and your contributions.

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md [hunk 1]
```diff
@@ -294,7 +294,11 @@ Accept wildcard characters: False
 ```
 
 ### -EnableHTTPRedirect
-Indicates that this cmdlet enables HTTP redirect for Web Application Proxy.
+Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication. Here is an example in PowerShell to create such a rule:
```
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication. 
```

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md [hunk 2]
```diff
@@ -294,7 +294,11 @@ Accept wildcard characters: False
 ```
 
 ### -EnableHTTPRedirect
-Indicates that this cmdlet enables HTTP redirect for Web Application Proxy.
+Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication. Here is an example in PowerShell to create such a rule:   
+
```
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
```

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md (hunk 1 above)
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication.
```

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md (hunk 2 above)
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
```

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md [hunk 3]
```diff
@@ -294,7 +294,11 @@ Accept wildcard characters: False
...
+```
```
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
```

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md [hunk 4]
```diff
@@ -294,7 +294,11 @@ Accept wildcard characters: False
 ```
 
 ### -EnableHTTPRedirect
-Indicates that this cmdlet enables HTTP redirect for Web Application Proxy.
+Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication. Here is an example in PowerShell to create such a rule:   
+
+```  
+New-NetFirewallRule -DisplayName "Allow 80 to 443 redirection" -Name "80to443redirect" -Action Allow -Direction Inbound -Protocol TCP -LocalPort 80
```
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
```

File: docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md [hunk 5]
```diff
@@ -294,7 +294,11 @@ Accept wildcard characters: False
 ```
 
 ### -EnableHTTPRedirect
-Indicates that this cmdlet enables HTTP redirect for Web Application Proxy.
+Indicates that this cmdlet enables HTTP redirect for Web Application Proxy. Note that you will need to allow the traffic on port TCP 80 on the WAP servers expecting to use this publication. Here is an example in PowerShell to create such a rule:   
+
+```  
+New-NetFirewallRule -DisplayName "Allow 80 to 443 redirection" -Name "80to443redirect" -Action Allow -Direction Inbound -Protocol TCP -LocalPort 80  
+```
```
Comment (nenonix) [inline docset/windows/webapplicationproxy/Add-WebApplicationProxyApplication.md]:
```suggestion
```

**Comment by piaudonn** [comment]
I am not sure how to do that :) can you break it down for me?

**Comment by nenonix** [comment]
@piaudonn After each of my suggested edits you can choose to accept them one by one - this will push the changes to your PR. Alternatively you can make the changes yourself if you edit your PR in your fork which is https://github.com/piaudonn/windows-powershell-docs/tree/patch-5. To get there click on **Files Changed** at the top of this page, where you should see the option to accept my changes, or (at top right) the option to **edit** the PR. I hope that makes sense 😄

**Comment by e0i** [comment]
@piaudonn Closing as there has been no follow-up with the reviewers' revision suggestions. You may re-open the PR to proceed with the review with **neonix**'s suggestions. Thank you for your patience with the process and your contributions.

**Pull request closed without merging**